src/PerformanceSensitiveAnalyzers/UnitTests/CallSiteImplicitAllocationAnalyzerTests.cs

```diff
@@ -161,6 +161,34 @@ public void Testing()
             await VerifyCS.VerifyAnalyzerAsync(sampleProgram);
         }
 
+        [Fact]
+        public async Task ParamsIsPrecededByOptionalParameters()
```

Comment:
Can you do something equivalent for vbnet?

Reply:
@Evangelink This is a C# only analyzer.
I think we should file an issue to support VB for it. It seems reasonable to be applicable to both languages.

Reply:
It should be straight forward to convert this to a language agnostic analyzer, since it's already based on IOperation. Could you please file an issue to track it? Thanks!